docusaurus/docs/iOS/customization/localization.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Localization
+---
+
+StreamChat offers pre-defined localization for text indicating across the whole UISDK. Currently, StreamChat supports only English language, but if you want to provide your custom strings, it's very easy to do.
+The whole approach to StreamChat SDK is to stay as much consistent as possible with native iOS paradigms and approaches to certain problems. That's why we use `strings` and `stringsdict` files to handle localize the framework.
+For advanced plural localization cases (typing indicator showing how many people are typing or how many responses are in thread), we use `stringsdict` with `NSStringPluralRuleType` 
+
+## Adding Custom Localization
+
+As said earlier, StreamChatUI uses `strings` and `stringsdict` files to handle localize the framework.
+`Appearance` structure contains closure `localizationProvider`, which is responsible for deriving `localizedString` from exact bundle. 
+By default, the bundle is  `StreamChatUI` framework bundle, but you can customize it to anything you need in 3 steps:
+
+1. First thing you need to do is to copy `StreamChatUI` localization keys into your `strings` file (Or just copy the default `Localizable.strings` and `Localizable.stringsdict` files to your project from our repository).
+2. Once you have those files in your application, you need to set the `localizationProvider` to provide your `AppBundle` instead of `StreamChatUI` frameworks like this: 
+```swift
+Appearance.default.localizationProvider = { key, table in
+    // For most cases, you will probably use Bundle.main, unless your strings are in different Bundle.
+    // Note: `table` is equivalent to filename of the `Localization.strings` file without file extension. 
+    Bundle.main.localizedString(forKey: key, value: nil, table: table)
+}
+```
+3. Now everything that has left is to implement your  `strings` and `stringsdict`  files for different languages or even customize the default strings for your own.
```

Comment:
- how should you name strings so that are linked to a specific language?
- how do you get the app to use a different language other than english?